src/app.ts

```diff
@@ -170,9 +191,13 @@ const processPushEvent = async (payload: PushEvent, context: Context<'push'>) =>
   const filesChanged = await getCommitFiles(repository, payload.after)
   if (!filesChanged.includes(configFileName)) return
 
-  const parsed = await determineConfigurationChanges(configFileName, repository, payload.after)
+  const either = await determineConfigurationChanges(configFileName, repository, payload.after)
 
-  if (!parsed.repositoryConfiguration) return
+  if (E.isLeft(either)) {
+    log.error('There has been an error while processing event %o', either.left)
+    return
+  }
+  const parsed = either.right
```

Comment:
```suggestion
  if (E.isLeft(parsed)) {
    return parsed
  }
```

Coukdn't we simplify like this? 😄  Is there a need to error log, doesn't that already happen? Do we need to error log for validation errors?

Reply:
In this case I think so, because this happens on `processPushEvent`. It would be good to have some logging to know why `processPushEvent` exit, specially since this should never happen.